trampoline/src/main/java/org/ernest/applications/trampoline/services/FileManager.java

```diff
@@ -38,6 +44,14 @@
     @Value("${trampoline.version}")
     private float currentVersion;
 
+    private GradleRunner gradleRunner;
+    private MavenRunner mavenRunner;
+
+    public FileManager() {
+    	gradleRunner = new GradleRunner();
+    	mavenRunner = new MavenRunner();
+	}
+

```

Comment:
Would not be better to set both runners new Objects Instances directly on runMicroservice method rather than being attributes on FileManager?

Reply:
I thought about that, but only one instance is needed.